src/views/Home/components/chapelProgress/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Doughnut } from 'react-chartjs-2';
+import Card, { CardContent } from 'material-ui/Card';
+
+import { gordonColors } from '../../../../theme';
+import user from '../../../../services/user';
+import GordonLoader from '../../../../components/Loader';
+
+export default class ChapelPsrogress extends Component {
+  constructor(props) {
+    super(props);
+
+    this.loadChapel = this.loadChapel.bind(this);
+
+    this.state = {
+      loading: true,
+      chapelCredits: {},
+    };
+  }
+  componentWillMount() {
+    this.loadChapel();
+  }
+  async loadChapel() {
+    const chapelCredits = await user.getChapelCredits();
+    this.setState({ loading: false });
+    this.setState({ chapelCredits });
+  }
+  render() {
+    let content;
+    const { current, required } = (this.state.chapelCredits);
+    const remaining = required - current;
+    const Data = {
+      datasets: [{
+        data: [current, remaining],
+        backgroundColor: [gordonColors.primary.blue],
+      }],
+      labels: [
+        'Chapel Events Attended',
+        'Events Remaining',
+      ],
+    };
+    if (this.state.loading === true) {
+      content = <GordonLoader />;
```

Comment:
With this approach (in this component and the other one), the homepage just looks like two random loading spinners until the data has arrived.

A better approach would be to swap out the `<Doughnut ... />` element for a loader, then replace the loader with the chart once the data has loaded. This way, there would be context for what the user is waiting for ("oh, my chapel credits are loading"), and the card would be around the loader, so it would look better.

As a result, this would look like:
```JavaScript
    if (this.state.loading === true) {
      content = <GordonLoader />;
    } else {
      content = (
        <figure>
          <figcaption>
            <h4>{current}/{required} Attended Chapels Events</h4>
          </figcaption>
          <Doughnut data={Data} />
        </figure>
      );
    }
    return (
      <Card>
        <CardContent>
          <h3>Chapel Progress</h3>
          { content }
        </CardContent >
      </Card>
    );
```